Assets/SteamVR_Unity_Toolkit/Scripts/VRTK_BezierPointer.cs

```diff
@@ -37,6 +40,13 @@ public class VRTK_BezierPointer : VRTK_WorldPointer
         private GameObject curvedBeamContainer;
         private CurveGenerator curvedBeam;
 
+        // materials of customPointerCursor and teleportBeam (if defined)
+        private Material customPointerMaterial;
+        private Material beamTraceMaterial;
+        [Tooltip("Adapt tracer instances to the curve length")]
+        [SerializeField]
+        private bool rescalePointerTracer = false;
```

Comment:
For me it is still not clear when it is better to expose a field with SerializeField or define it as public. The point here is that this field should be defined before the initialization of the pointer, so my intention was to prevent changing it at runtime (without a proper method or property defined). What do you suggest do do?